examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -538,14 +538,6 @@
     "We are now ready to train our model."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    get_booking('/workspace/data')
</pre><p>This does data preprocessing under the hood as well, therefore, it is time and mostly memory consuming. is it possible that you only download the raw dataset since you are doing the preprocessing already in the following cells?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/943/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I am quite fond of relying on the functionality in our library to get the data, makes the notebooks much cleaner (fewer lines of code) and introduces users to the functionality of our library, but happy to change this if that is what we would like to do here